### PR TITLE
Feature/revise login flow

### PR DESCRIPTION
**Pull request by froggomad**
# [User Story](HTTPS://TRELLO.COM) 

## Changes
- Revised flow so login goes to survey storyboard
- Separated member list from member detail storyboard
- Added reference to profile storyboard on survey TabBarViewController

## Checklist
`choose one minimum`
- [ ] created unit test
- [ ] updated unit test
- [ ] created UI test
- [ ] updated UI test
- [x] bug fix/other

`required`
- [x] tested locally
- [ ] updated the docs (https://github.com/Lambda-School-Labs/Labs26-Apollo-iOS-TeamA/blob/master/.github/documentation_standards.md)

`optional`
- [ ] added new dependencies

## Screenshots

Survey Storyboard with User Profile in tab bar

Member Detail Storyboard without Member list

Member List Storyboard

Survey Storyboard with reference

### CODE REVIEW

File: labs-ios-starter/Application/Extensions/String + identifier.swift [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+enum TableViewIdentifier: String {
+    case profileCell
+}
+
+extension String {
+    ///convenience method to retrieve Storyboard Identifiers
+    static func getStoryboardIdentifier(identifier: StoryboardIdentifier) -> String {
+        //storyboard ids are capitalized
+        identifier.rawValue.firstCapitalized
```
Comment (Shawn-James) [inline labs-ios-starter/Application/Extensions/String + identifier.swift:33]: Consider having the cases themselves be capitalized, or their raw values be capitalized, and then calling the enum directly. Or if you wanted this to be called like a string, maybe you could extend string itself to include the StoryboardIdentifier enum. Would be nice to access this value with built in getter, unless I’m missing something here.

File: labs-ios-starter/Application/Network/ErrorHandler.swift [hunk 2]
```diff
@@ -64,6 +66,7 @@ class ErrorHandler {
             NetworkError.resourceNotAcceptable.rawValue: "Resource not acceptable",
             NetworkError.tooManyRequests.rawValue: "Too many requests sent recently",
             NetworkError.headerFieldTooLarge.rawValue: "header too large",
+            NetworkError.internalServerError.rawValue: "An internal server error occurred",
             NetworkError.unknown.rawValue: "An unknown error occured"
         ]
     }()
```
Comment (Shawn-James) [inline labs-ios-starter/Application/Network/ErrorHandler.swift:72]: This is great

File: labs-ios-starter/Login/View Controllers/LoginViewController.swift [hunk 3]
```diff
@@ -62,17 +62,17 @@ class LoginViewController: DefaultViewController {
                 self.presentedViewController == nil else { return }
                 
             if exists {
-                self.performSegue(withIdentifier: "ShowDetailProfileList", sender: nil)
+                self.performSegue(withIdentifier: .getSegueIdentifier(identifier: .showDetailProfileList), sender: nil)
```
Comment (Shawn-James) [inline labs-ios-starter/Login/View Controllers/LoginViewController.swift]: Would be nice to shorten this up. I wonder if you can put the enum directly inside the string extension and use the built-in getter

Comment (froggomad) [reply labs-ios-starter/Login/View Controllers/LoginViewController.swift]: we could just do as you suggested in a previous comment and use the enum with Raw values. I was trying to eliminate us having to remember to store the raw value separately. Both approaches have their unique drawbacks and benefits. We could also make static properties on String using the convenience method to capitalize names. I.e. `static var someIdentifier: String = .getSegueIdentifier(.someIdentifier)` the we'd have `.someIdentifier` to use. But it's an extra step, so might as well use the enum lol

File: labs-ios-starter/Profiles/View Controllers/ProfileDetailViewController.swift [hunk 4]
```diff
@@ -13,11 +13,13 @@ class ProfileDetailViewController: UIViewController {
     // MARK: - Properties and Outlets
     
     @IBOutlet weak var avatarImageView: UIImageView!
-    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var firstNameLabel: UILabel!
+    @IBOutlet weak var lastNameLabel: UILabel!
     @IBOutlet weak var emailLabel: UILabel!
```
Comment (Shawn-James) [inline labs-ios-starter/Profiles/View Controllers/ProfileDetailViewController.swift:19]: The organization is clearly grouped mostly by type here. The imageView and stackView throw me off and seem random. Maybe they could be in their own sections or something if group by type is the goal here.

Comment (froggomad) [reply labs-ios-starter/Profiles/View Controllers/ProfileDetailViewController.swift:19]: I normally try to group outlets in the order they appear on the page from left->right, top->bottom and all in one section. Happy to change it so its grouped by type if that makes more sense to you

**Review by Shawn-James — APPROVED** [review]

File: labs-ios-starter/Profiles/View Controllers/ProfileDetailViewController.swift (hunk 4 above)
Comment (froggomad) [reply labs-ios-starter/Profiles/View Controllers/ProfileDetailViewController.swift:19]: as far as the stackview, it's listed with the views contained in it

Comment (Shawn-James) [reply labs-ios-starter/Profiles/View Controllers/ProfileDetailViewController.swift:19]: Ah okay I see it, that’s actually a more common way to do it. And then it looks like things are grouped based on proximity. Good as is then.

File: labs-ios-starter/Login/View Controllers/LoginViewController.swift (hunk 3 above)
Comment (froggomad) [reply labs-ios-starter/Login/View Controllers/LoginViewController.swift]: Just thinking about this more, another downside to just using the rawValue is you lose access to calling it directly (You would have to call `.Identifier.someIdentifier.rawValue`) - that seems as long

Comment (froggomad) [reply labs-ios-starter/Login/View Controllers/LoginViewController.swift]: I could (should?) make `identifier:` unnamed - and call the method `getID`

Comment (Shawn-James) [reply labs-ios-starter/Login/View Controllers/LoginViewController.swift]: Right. Hmm. I’m going to clone and play with this for a second.

**Pull request merged**